plugins/destination/sqlite/docs/overview.md

```diff
@@ -3,21 +3,12 @@ name: SQLite
 stage: GA
 title: SQLite Destination Plugin
 description: CloudQuery SQLite destination plugin documentation
+hub-title: SQLite Destination Plugin
```

Comment:
What's wrong with `Overview` in the sidebar? `SQLite Destination Plugin` will be quite long I think

<img width="828" height="278" alt="image" src="https://github.com/user-attachments/assets/fca0b9a6-04e6-497a-8aad-96016ee7dcb8" />